src/Compute/Compute/help/New-AzVM.md

```diff
@@ -358,24 +353,6 @@ $vm = Get-AzVM -ResourceGroupName $rgname -Name $vmname;
 ```
 This example shows how to create a VM with a valid Gen2 image, allowing the VM to default to TrustedLaunch which requires Gen2 images. Please check [the Trusted Launch feature page](aka.ms/trustedlaunch) for more information.
 
-### Example 10: Creates a VM with TrustedLaunch turned on by default.
```

Comment:
This example seems helpful to keep, especially given how invasive the change to TL by Default was.

Reply:
With TL-as-default, our goal is to keep seamless experience for end users. i.e., they're creating VMs using usual cmdlets/scripts without change and now VM is foundationally secure with Trusted launch.

Hence, highlighting TL-as-default change is more invasive will conflict with this goal.